AyOK-Code/oscn
# tables made for tulsa city, scraper in_progress

### To Test

Open three Command line tabs.

run `redis-server` in one

run ` bundle exec sidekiq` in one

after these are running run `bundle exec rake tulsa_city:inmate_list` in the last tab.  

Inmate record count should equal Inmate_json count( which is used for the progress bar)
